components/onboarding/Education.tsx

```
@@ -89,39 +90,56 @@ const Education = (): JSX.Element => {
+
+										{education.finished_at_month && education.finished_at_year && ' until '}
+										{education.finished_at_month && months.find(month => parseInt(month.value) === education.finished_at_month)?.label} {education.finished_at_year?.toString()}
+									</p>
+
+									{/* Expandable description section */}
+									{education.description && (
+										<div className="mt-3 max-h-0 opacity-0 overflow-hidden transition-all duration-300 ease-in-out group-hover:max-h-48 group-hover:opacity-100">
+											<div className="border-t border-gray-200">
+												<div
+													className="prose prose-sm max-w-none text-gray-700 max-h-44 overflow-hidden"
+													dangerouslySetInnerHTML={{__html: education.description}}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)